src/altimate.ts

```diff
@@ -207,22 +207,22 @@ export class AltimateRequest {
   constructor(
     private telemetry: TelemetryService,
     private dbtTerminal: DBTTerminal,
+    private pythonEnvironment: PythonEnvironment,
   ) {}
 
-  getConfig(): AltimateConfig | undefined {
-    const key = workspace.getConfiguration("dbt").get<string>("altimateAiKey");
-    const instance = workspace
-      .getConfiguration("dbt")
-      .get<string>("altimateInstanceName");
+  getInstanceName() {
```

Comment:
I think since getting API key is not simple anymore as we are doing env var substitutions, we should have only one way getting them. And in some places we need them separately. WDYT

Reply:
I would think they only make sense if they are together and if you have not one of them you don't have a config.